modules/ensemble/lib/widget/input/dropdown.dart

```diff
@@ -439,6 +498,61 @@ class SelectOneState extends FormFieldWidgetState<SelectOne>
                         : null,
                     fillColor: widget._controller
                         .fillColor, // Background color for the field
+                    enabledBorder: widget._controller.dropdownBorderColor != null ||
```

Comment:
the autocomplete enabledBorder code seems like a copy paste of the one before, please re-use by creating a function instead of copy/pasting

Reply:
Created.